postgres/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP DATABASE IF EXISTS reviews
+
+CREATE DATABASE reviews
+
+CREATE TABLE reviews.listings
+(
+  listing_id INTEGER UNIQUE NOT NULL,
+  PRIMARY KEY(listing_id),
```

Comment:
Hi Elana! line 7 looks a bit excess to me, as far as I know if you set listing_id to primary key it's already unique and not null
https://www.postgresqltutorial.com/postgresql-primary-key/

Reply:
@annkaz just updated to update all the PRIMARY KEYS